app-engine/java/maven/src/com.google.cloud.tools.intellij.appengine.java.maven/MavenAppEngineWebXmlDirectoryProvider.java

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I think the package directory path got named incorrectly with '.' instead of as a bunch of sub folders.

Reply:
oh wow good catch

Reply:
package structure should be fixed